tests/lib/custom_lib/src/main.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @file test custom_lib library
+ *
+ * This suite verifies that the methods provided with the custom_lib
+ * library works correctly.
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <custom_lib.h>
+
+static void test_get_value(void)
+{
+	int values[] = { INT_MIN, INT_MIN + 1, -1, 0,
+			 1, INT_MAX - 1, INT_MAX };
+
+	/* Verify standard behavior */
+	for (int i = 0; i < sizeof(values)/sizeof(values[0]); ++i) {
```

Comment:
It would be more idiomatic to use ARRAY_SIZE from zephyr.h.

Reply:
Done.